src/SourceBuild/content/Directory.Build.props

```diff
@@ -190,6 +190,7 @@
     <PoisonReportDataFile>$(PackageReportDir)poison-catalog.xml</PoisonReportDataFile>
     <PoisonedReportFile>$(PackageReportDir)poisoned.txt</PoisonedReportFile>
     <PoisonUsageReportFile>$(PackageReportDir)poison-usage.xml</PoisonUsageReportFile>
+    <DISABLE_CROSSGEN>true</DISABLE_CROSSGEN>
```

Comment:
Does this work? I think this would need to be defined as an env var when building the sdk repository. But aren't there are also other repositories that invoke crossgen?

Reply:
I took this prop from [here](https://github.com/dotnet/dotnet/blob/67e54850463002d2502780a8a6058206d44c02be/src/sdk/src/Installer/redist-installer/targets/Crossgen.targets#L5) and [here](https://github.com/dotnet/dotnet/blob/67e54850463002d2502780a8a6058206d44c02be/src/sdk/eng/DotNetBuild.props#L18) in sdk. When this issue was first discovered, setting this property to true resulted in newly detected poison leaks.

A quick investigation shows that crossgen is used in [runtime](https://github.com/dotnet/dotnet/blob/e01b3fa3c490b78d0c87c48a2fe418e9d2d99407/src/runtime/src/tasks/Crossgen2Tasks/Microsoft.NET.CrossGen.targets#L134) (for example). I think the question I'm having trouble answering now is whether this is something that needs to be disabled. To me, https://github.com/dotnet/dotnet/blob/67e54850463002d2502780a8a6058206d44c02be/src/sdk/src/Installer/redist-installer/targets/Crossgen.targets#L5 is a clear example of crossgenning assemblies we use during the build. I can't say if the runtime example is doing the same?

Reply:
OK. FWIW this property doesn't flow into dotnet/sdk. It's just set in the VMR orchestrator and never gets passed to dotnet/sdk's build.

Reply:
![image](https://github.com/user-attachments/assets/d53b6215-492f-4121-8bb7-5b20eb614ec7)
You were right about it not getting passed. I moved the prop to the sdk.proj file and now crossgen has been disabled as expected. Thanks for catching that.